src/utils/app/share.ts

```diff
@@ -57,7 +57,7 @@ export const getUnpublishActionByType = (type: SharingType) => {
   }
 };
 
-export const isEntityExternal = (entity: ShareEntity) =>
+export const isThisEntityExternal = (entity: ShareEntity) =>
```

Comment:
What is the difference between isThisEntityExternal and isExternalEntity? 
It's not clear when to use them
Is it possible to rename somehow?

Reply:
Maybe it's possible to use full check for external everywhere?

Reply:
@mikitabut , I specially added `This` word to have difference when we check entity itself only (and when we check entity and parents)